### PR TITLE
[13.0] base_jsonify: fix rounding for float fields

### PR DESCRIPTION
**Pull request by i-vyshnevska**
use precision defined on fields for data preparing 
before data "13.95" returned as "13.950000000000001"

### CODE REVIEW

**Comment by i-vyshnevska** [comment]
side effect now it returned as a string

**Comment by i-vyshnevska** [comment]
@simahawk can you give an opinion on this change

File: base_jsonify/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {
     "name": "Base Jsonify",
     "summary": "Base module that provide the jsonify method on all models",
-    "version": "13.0.1.3.0",
+    "version": "13.0.1.4.0",
```
Comment (simahawk) [inline base_jsonify/__manifest__.py]: rollback this pls, we manage it via ocabot. In any case this seems more a patch than an improvement :wink:

**Review by simahawk — CHANGES_REQUESTED** [review]
can you re-word the commit msg to include module name (as per PR title)?

**Comment by simahawk** [comment]
> side effect now it returned as a string

this is a problem. It will change the behavior in maaaany places.
I'd make this conditional or convert it back to number.

**Comment by i-vyshnevska** [comment]
@simahawk updated

**Review by lmignon — CHANGES_REQUESTED** [review]
Thank you @i-vyshnevska Code LGTM but can you add a tests for this change PLZ.

**Comment by jcoux** [comment]
@i-vyshnevska Fix after your rebase: https://github.com/i-vyshnevska/server-tools/pull/1

**Review by ivantodorovich — APPROVED** [review]

**Review by ajaniszewska-dev — APPROVED** [review]

**Pull request closed without merging**